core/contracts/financial-templates/expiring-multiparty/Liquidatable.sol

```diff
@@ -295,7 +301,8 @@ contract Liquidatable is PricelessPositionManager {
             liquidationId,
             tokensLiquidated.rawValue,
             lockedCollateral.rawValue,
-            liquidatedCollateral.rawValue
+            liquidatedCollateral.rawValue,
+            getCurrentTime()
```

Comment:
@mrice32 @chrismaree @kendricktan Can someone else confirm that asking for the the `now` blocktime would not be different during a single function call? Note that `getCurrentTime()` returns `now` and is used three times in this single contract method.

Reply:
i just want to say i did try to create a new local var since getCurrentTime is called multiple times within this function, but this caused compiler stack depth error.  calling the function again in the event was how i got around this.

Reply:
I am confident that `now` (and `getCurrentTime()` even with simulated time) will always return the same time within the same transaction.